cmd/buildah/pull.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+	"github.com/projectatomic/buildah/pkg/parse"
+	util "github.com/projectatomic/buildah/util"
+	"github.com/urfave/cli"
+)
+
+var (
+	pullFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "authfile",
+			Usage: "path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
+		},
+		cli.StringFlag{
+			Name:  "cert-dir",
+			Value: "",
+			Usage: "use certificates at the specified path to access the registry",
+		},
+		cli.StringFlag{
+			Name:  "creds",
+			Value: "",
+			Usage: "use `[username[:password]]` for accessing the registry",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "don't output progress information when pulling images",
+		},
+		cli.StringFlag{
+			Name:  "shm-size",
+			Usage: "size of '/dev/shm'. The format is `<number><unit>`.",
+			Value: "65536k",
+		},
+		cli.StringFlag{
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
+		},
+		cli.BoolTFlag{
+			Name:  "tls-verify",
+			Usage: "require HTTPS and verify certificates when accessing the registry",
+		},
+	}
+	pullDescription = "Pulls an image from the specified location"
+
+	pullCommand = cli.Command{
+		Name:           "pull",
+		Usage:          "Pull an image from the specified location",
+		Description:    pullDescription,
+		Flags:          append(pullFlags),
+		Action:         pullCmd,
+		ArgsUsage:      "IMAGE",
+		SkipArgReorder: true,
+	}
+)
+
+func pullCmd(c *cli.Context) error {
+	args := c.Args()
+	if len(args) == 0 {
+		return errors.Errorf("an image name must be specified")
+	}
+	if len(args) > 1 {
```

Comment:
Should we allow the specification of multiple images?

Reply:
I was keeping step with 'buildah from' and it looks like podman pull only does one image at a time.  Can we leave it as one for now and come back and revisit?